ltp-production.yaml

```diff
@@ -1,467 +1,483 @@
-    - lkft/linux-stable-rc-4.14-oe
-    - lkft/linux-stable-rc-4.16-oe
-    - lkft/linux-stable-rc-4.17-oe
-    - lkft/linux-mainline-oe
-    test_name: ltp-cve-tests/cve-2016-7117
-    url: https://bugs.linaro.org/show_bug.cgi?id=3884
+
+  - environments: *environments_all
+    notes: >
+      LKFT: LTP open posix: pthread_rwlock_rdlock_2-1 and
+      pthread_rwlock_rdlock_2-2 failed
+    projects: *projects_all
+    test_name: ltp-open-posix-tests/pthread_rwlock_rdlock_2-1
+    url: https://bugs.linaro.org/show_bug.cgi?id=3952
     active: true
     intermittent: false
-  - environments:
-    - slug: qemu_arm
-    notes: 'LKFT: qemu_arm: LTP CVE cve-2015-7550 Test timeouted, sending SIGKILL!'
-    projects:
-    - lkft/linux-stable-rc-4.4-oe
-    - lkft/linux-stable-rc-4.9-oe
-    - lkft/linux-stable-rc-4.14-oe
-    - lkft/linux-stable-rc-4.16-oe
-    - lkft/linux-stable-rc-4.17-oe
-    - lkft/linux-mainline-oe
-    test_name: ltp-cve-tests/cve-2015-7550
-    url: https://bugs.linaro.org/show_bug.cgi?id=3883
+  - environments: *environments_all
+    notes: >
+      LKFT: LTP open posix: pthread_rwlock_rdlock_2-2 and
+      pthread_rwlock_rdlock_2-2 failed
+    projects: *projects_all
+    test_name: ltp-open-posix-tests/pthread_rwlock_rdlock_2-2
+    url: https://bugs.linaro.org/show_bug.cgi?id=3952
     active: true
     intermittent: false
-  - environments:
-    - slug: qemu_arm
-    notes: 'LKFT: qemu_arm: LTP CVE cve-2015-7550 Test timeouted, sending SIGKILL!'
-    projects:
-    - lkft/linux-stable-rc-4.4-oe
-    - lkft/linux-stable-rc-4.9-oe
-    - lkft/linux-stable-rc-4.14-oe
-    - lkft/linux-stable-rc-4.16-oe
-    - lkft/linux-stable-rc-4.17-oe
-    - lkft/linux-mainline-oe
-    test_name: ltp-syscalls-tests/keyctl02
-    url: https://bugs.linaro.org/show_bug.cgi?id=3883
+
+  - environments: *environments_all
+    notes: >
+      LKFT: LTP: open posix mmap_11-4 : Modification of the partial page at the
+      end of an object is written out
+    projects: *projects_all
+    test_name: ltp-open-posix-tests/mmap_11-4
+    url: https://bugs.linaro.org/show_bug.cgi?id=3953
     active: true
     intermittent: false
-  - environments:
-    - slug: qemu_arm64
-    - slug: qemu_arm
-    notes: 'qemu_arm32/64: LTP select04 is not returning 0 on timeout'
-    projects:
-    - lkft/linux-stable-rc-4.4-oe
-    - lkft/linux-stable-rc-4.9-oe
-    - lkft/linux-stable-rc-4.14-oe
-    - lkft/linux-stable-rc-4.16-oe
-    - lkft/linux-stable-rc-4.17-oe
-    - lkft/linux-mainline-oe
-    test_name: ltp-syscalls-tests/select04
-    url: https://bugs.linaro.org/show_bug.cgi?id=3852
+
+  - environments: *environments_all
+    notes: >
+      LKFT: LTP: open posix: clock_settime_8-1 intermittent failures on all
+      device
+    projects: *projects_all
+    test_name: ltp-open-posix-tests/clock_settime_8-1
+    url: https://bugs.linaro.org/show_bug.cgi?id=3965
     active: true
-    intermittent: false
+    intermittent: true
+
+#  - environments:
```

Comment:
delete commented part? If it's not used carrying it makes little sense to me.

Reply:
I wanted to preserve the work you did to bring over the content from the skiplists. I moved the other files into unused/, but the data will be used in the next couple weeks as we fill out all of the known issues.

In other words, it's temporary and useful for the moment.